.github/workflows/build.yml

```diff
@@ -26,10 +26,6 @@ jobs:
     runs-on: ${{ matrix.os }}
     timeout-minutes: 60
```

Comment:
With the mutation tests moved out, we can probably reduce this to 20 minutes.

Reply:
Thanks. Updated.